### PR TITLE
Set the fixer name statically so that cs-fixer and import

### PR DESCRIPTION
**Pull request by mdk999**

### CODE REVIEW

File: src/Fixer/AbstractSymplifyFixer.php [hunk 1]
```diff
@@ -20,7 +20,7 @@ public function getPriority(): int
 
     public function getName(): string
     {
-        return static::class;
+        return 'Nocworx/codingstandard';
```
Comment (mclorch) [inline src/Fixer/AbstractSymplifyFixer.php:23]: what's the reason for this change?

Comment (mclorch) [reply src/Fixer/AbstractSymplifyFixer.php:23]: e.g. what was the behavior prior to this change vs with this applied

Comment (mclorch) [reply src/Fixer/AbstractSymplifyFixer.php:23]: Michael confirmed that there would be an error if this is not hard-coded

**Review by mpedrummer — APPROVED** [review]

**Review by ppf — APPROVED** [review]
